### ISSUE TITLE
babili issue with pino

### ISSUE THREAD

**Issue by ravvpn** [babel/babili#547]
I was using version `0.0.12` of `babili` to `webpack` my app along with the  [pino](https://www.npmjs.com/package/pino) module which worked perfectly - however, after upgrading `babili` to `0.1.1` I am now getting an error when running `webpack`

app dependencies:
* webpack 2.6.0
* babel-core 6.24.1
* babel-loader 7.0.0
* babel-preset-babili 0.1.1 (upgraded from 0.0.12)
* babel-preset-env 1.5.1
* babel-polyfill 6.23.0
* pino 4.5.2

the error:
```
ERROR in ../~/pino/pino.js
Module build failed: Error: ~/app/node_modules/pino/pino.js: Binding not found for BindingIdentifier. onTerminatedPlease report this at https://github.com/babel/babili/issues/new
    at BindingIdentifier ~/app/node_modules/babel-plugin-minify-mangle-names/lib/index.js:216:21)
    at newFn (~/app/node_modules/babel-traverse/lib/visitors.js:318:17)
    at bfsTraverse (~/app/node_modules/babel-plugin-minify-mangle-names/lib/bfs-traverse.js:34:43)
    at Mangler.collect (~/app/node_modules/babel-plugin-minify-mangle-names/lib/index.js:260:9)
    at Mangler.run (~/app/node_modules/babel-plugin-minify-mangle-names/lib/index.js:70:14)
    at PluginPass.exit ~/app/node_modules/babel-plugin-minify-mangle-names/lib/index.js:580:19)
    at newFn ~/app/node_modules/babel-traverse/lib/visitors.js:276:21)
    at NodePath._call (~/app/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (~/app/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit ~/app/node_modules/babel-traverse/lib/path/context.js:117:8)
    at TraversalContext.visitQueue (~/app/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitSingle (~/app/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit ~/app/node_modules/babel-traverse/lib/context.js:192:19)
    at Function.traverse.node (~/app/node_modules/babel-traverse/lib/index.js:114:17)
    at traverse (~/app/node_modules/babel-traverse/lib/index.js:79:12)
    at File.transform (~/app/node_modules/babel-core/lib/transformation/file/index.js:548:35)
    at ~/app/node_modules/babel-core/lib/transformation/pipeline.js:50:19
    at File.wrap (~/app/node_modules/babel-core/lib/transformation/file/index.js:564:16)
    at Pipeline.transform (~/app/node_modules/babel-core/lib/transformation/pipeline.js:47:17)
    at transpile (~/app/node_modules/babel-loader/lib/index.js:48:20)
    at Object.module.exports (~/app/node_modules/babel-loader/lib/index.js:163:20)
 @ ./app.js 1:81-96
 @ ./index.js
 @ multi babel-polyfill ./index.js
```

Please could you look into why this has occurred and release a bugfix or help me to resolve the issue. many thanks.

**Comment by boopathi** [comment]
Thanks. This is a mangler bug. I'll take a look at this.

Would be great if you can reproduce the error for a small snippet. Looks like there is some assignment/update/decl operation using the variable `onTerminated` and it's happening for some code like -

```js
onTerminated = foo;
onTerminated++;
var onTerminated = bar;
// etc...
```

**Comment by tomkel** [comment]
I am getting several `Binding not found for BindingIdentifier` errors as well.

**Comment by boopathi** [comment]
1. Can you post the code where you're getting this error ?
1. Are you using babili in `.babelrc` file or are you using `gulp-babili` or `babili-webpack-plugin` ? If `.babelrc` are you using es2015 preset or env preset with settings that convert `let/const` to `var`

**Comment by boopathi** [comment]
Released in [0.1.2](https://github.com/babel/babili/releases/tag/babili%400.1.2)
